File: Python/Practise_Files/gcd.py
def gcd(a, b):
    while b != 0:
        a, b = b, a % b  # Replace a with b and b with a % b (remainder)
    return a


def main():
    a, b = map(int, input("Enter two numbers: ").strip().split())
    print(f"The GCD of the numbers is {gcd(a,b)}.")


if __name__ == "__main__":
    main()

File: C_Programing/Practise_Programs/College/gcd.c
#include<stdio.h>

int main(){
    int a,b, gcd;
    printf("Enter two numbers: ");
    scanf("%d %d", &a, &b);

    for(int i=1;(i<=a && i<=b);i++){
        if(a%i==0 && b%i==0){
            gcd = i;
        }
    }
    printf("The gcd of %d and %d is %d\n", a, b, gcd);
    return 0;
}